ste/sourceInfoProvider-Local_windows.go

```diff
@@ -0,0 +1,28 @@
+// +build windows
+
+package ste
+
+import (
+	"golang.org/x/sys/windows"
+
+	"github.com/Azure/azure-storage-azcopy/sddl"
+)
+
+// This file os-triggers the ISDDLBearingSourceInfoProvider interface on a local SIP.
+
+func (f localFileSourceInfoProvider) GetSDDL() (string, error) {
+	// We only need Owner, Group, and DACLs for azure files.
+	sd, err := windows.GetNamedSecurityInfo(f.jptm.Info().Source, windows.SE_FILE_OBJECT, windows.OWNER_SECURITY_INFORMATION|windows.GROUP_SECURITY_INFORMATION|windows.DACL_SECURITY_INFORMATION)
```

Comment:
We do not.

During our original meeting with the Azure Files team, this was a question I asked them (since even obtaining SACLs require a higher permission level than azcopy would have from an average userland run)